tests/csm/rest/test_iam_users.py

```diff
@@ -4939,15 +4944,18 @@ def test_42284(self):
                                  "Entries not returned as expected")
         user_index = self.csm_obj.random_gen.randrange(1, count)
         marker = get_user_list[user_index]
-
+        self.log.info("User index is %s ", user_index)
         self.log.info("Step 3: Send GET request with max_entries as 15 and "
                       "marker as in between user")
         resp = self.csm_obj.list_iam_users_rgw(max_entries=15, marker=marker)
         assert_utils.assert_equals(resp.status_code, HTTPStatus.OK, "Status check failed")
         count_new = resp.json()["count"]
         get_user_list = resp.json()["users"]
-        actual_entries = self.csm_conf["common"]["num_users"] - user_index + 1
-        assert_utils.assert_equals(count_new, actual_entries, "Entries not returned as expected")
+        if len(resp1.json()["users"]) > 11:
+            assert_utils.assert_equals(count_new, 15, "Entries not returned as expected")
```

Comment:
11 & 15 are constant ?

Reply:
can we have these in test yaml or in other file

Reply:
sure, will move it to test config

Reply:
done